### PR TITLE
add page section and router section

### PR DESCRIPTION
**Pull request by shivamSspirit**
add page section and router section

### CODE REVIEW

File: src/routers/routes.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+
+import { BrowserRouter as Router, Route, Routes, Navigate } from "react-router-dom";
+
+
+
+function AllRoute() {
+    const [state, setState] = useState(true);
+
+// //  const navigate = useNavigate();
```
Comment (aurobindoGupta) [inline src/routers/routes.js:21]: remo0ve

**Review by aurobindoGupta — APPROVED** [review]
nice work.

File: src/pages/cart-page.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import Cart from '../components/cart/cart'
+import Header from '../components/header/header'
+
+function CartPage() {
+  return (
+    <div>
+      <Header/>
+      <Cart/>
+    </div>
```
Comment (viru2001) [inline src/pages/cart-page.js:10]: instead of `<div>` you can use React Frgaments here

**Review by viru2001 — APPROVED** [review]
Remove commented code.  
You should add `<Header />` component only once. There is no need add it on every page

File: src/routers/routes.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+                {/* <Routes> */}
+                <Route exact path="/" element={<HomePage />} />
+                <Route exact path="/auth/login" element={<LoginPage />} />
+                <Route exact path="/auth/signup" element={<SignupPage />} />
+                <Route exact path="/wishlist" element={<WishlistPage />} />
+                <Route exact path="/products/:id" element={<ProductListingPage />} />
+                <Route exact path="/cart" element={<CartPage />} />
+                <Route  path='/mockman' element={<Mockman/>} />
+            </Routes>
+{/*
```
Comment (MarkVed17) [inline src/routers/routes.js:40]: Remove these comments

**Review by MarkVed17 — APPROVED** [review]
Good to proceed!

**Pull request merged**